app/src/Model/Repository/Repository.php

```diff
@@ -1,33 +1,44 @@
 <?php
+
 namespace App\Model\Repository;
+
 use PDO;
 
 abstract class Repository
 {
-    protected $pdo;
-    protected $table;
-    protected $classManaged;
-    
-    public function __construct(PDO $pdo)
+	protected $pdo;
+	protected $table;
+	protected $classManaged;
+
+	public function __construct(PDO $pdo)
 	{
 		$this->pdo = $pdo;
-    }
-    
-    public function getList()
+	}
+
+	public function getList()
+	{
+		$q = $this->pdo->query('SELECT * FROM ' . $this->table);
+		$q->setFetchMode(PDO::FETCH_CLASS | PDO::FETCH_PROPS_LATE, $this->classManaged);
+
+		return $q->fetchAll();
+	}
+
+	public function getListOfActives()
 	{
-		$q = $this->pdo->query('SELECT * FROM '.$this->table);
+		$request = 'SELECT * FROM ' . $this->table . ' WHERE active = 1';
```

Comment:
active is specific to one entity ?
Maybe this method should be in the child class